hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieArchivedTimeline.java

```diff
@@ -248,11 +253,32 @@ private HoodieInstant readCommit(GenericRecord record, boolean loadDetails) {
               break;
             }
           }
+        } catch (Exception originalException) {
+          // merge small archive files may left uncompleted archive file which will cause exception.
+          // need to ignore this kind of exception here.
+          try {
+            Path planPath = new Path(metaClient.getArchivePath(), "mergeArchivePlan");
```

Comment:
Reuse `HoodieTimelineArchiveLog::mergeArchivePlanName`?

Reply:
We have to add ``public static final String MERGE_ARCHIVE_PLAN_NAME = "mergeArchivePlan";`` in `HoodieArchivedTimeline.java` and let `HoodieTimelineArchiveLog.java ` use it because of dependency issue.

Reply:
Got it.